src/front-end/typescript/lib/app/router.ts

```diff
@@ -111,10 +111,10 @@ const router: Router<Route> = {
     },
     {
       path: '/sign-in',
-      makeRoute() {
+      makeRoute({query}) {
         return {
           tag: 'signIn',
-          value: null
+          value: { redirectOnSuccess: query.redirectOnSuccess as string } // TODO(Jesse): Why does this need to be cast?
```

Comment:
In `framework.tsx`, the values of the query object can be `string | string[] | undefined`. You should use `getString(query, 'redirectOnSuccess') || ''` from `shared/lib`.